contrib/docker-compose/standalone/haproxy/etc/haproxy.cfg.tmpl

```diff
@@ -375,7 +385,7 @@ frontend fe_hockeypuck_ddos_lb
     log-format "%ci:%cp [%t] %ft %b/%s %Tq/%Tw/%Tc/%Tr/%Tt %ST %U/%B %CC %CS %tsc %ac/%fc/%bc/%sc/%rc %sq/%bq %hr %hs %{+Q}r"
 
     # Fetch stats from the primary Hockeypuck instance
-    use_backend be_hockeypuck_primary if keyserver_host get_pks_lookup pks_stats_query METH_GET
+    use_backend be_hockeypuck_primary if keyserver_host get_pks_lookup pks_op_stats METH_GET
```

Comment:
broken reference? Note that prometheus currently doesn't work because we don't have an equivalent of `proxy_http` in haproxy, which rewrites URL references bidirectionally, this will probably need to be solved by changing the prometheus config...

Reply:
I have copied a few more ACLs above and renamed `pks_stats_query` to `pks_op_stats`

Reply:
I wonder now, do we want to introduce a new ENV that will enable the restart behaviour?